prow/jobs/jinja/periodics/label_sync.jinja2

```diff
@@ -11,6 +11,13 @@
     containers:
     - name: label-sync
       image: gcr.io/k8s-prow/label_sync:v20221205-a1b0b85d88
+      resources:
+        limits:
+          cpu: 1
+        requests:
+          cpu: 1
+          memory: "500Mi"
+          ephemeral-storage: "100Gi"
```

Comment:
This is not something we want to request. In the system that was running for 2 years, we had 100Gi total for the entire node. There is no reason jobs should have access to 100Gi of local storage. I think we should just remove the `ephemeral-storage` request altogether.

Reply:
Removed `ephemeral-storage` from all requests